hw/databases/Postgresql.txt

```diff
@@ -0,0 +1,21 @@
+// Create table
+CREATE TABLE "person" (
+	id INT not NULL UNIQUE,
+	name VARCHAR(20) not NULL,
```

Comment:
Small note: if `NULL` in uppercase, than `not` should be too

Reply:
And I forgot to fix one more 'not' the first time: [634bed40](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/5/commits/634bed40c0c49009a42359f7ab16a5c30d122611)